### PR TITLE
Update container-registry-manage-artifact.md

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Remove redundant content from container-registry-manage-artifact.md

### CODE REVIEW

**Comment by Jak-MS** [comment]
@tejaswikolli-web 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by tejaswikolli-web — APPROVED** [review]

**Comment by tejaswikolli-web** [comment]
#sign-off

File: articles/container-registry/container-registry-manage-artifact.md [hunk 1]
```diff
@@ -56,36 +55,21 @@ Authenticate with the ACR, for allowing you to pull and push container images.
...
-
-If Docker isn't available, you can utilize the AD token provided for authentication. Authenticate with your [individual Microsoft Entra identity](container-registry-authentication.md?tabs=azure-cli#individual-login-with-azure-ad) using an AD token. Always use "000..." for the `USER_NAME` as the token is parsed through the `PASSWORD` variable.
-
-```azurecli
-# Login to Azure
-az login
 ```
 
-### Sign in with ORAS
+This setup enables you to seamlessly push and pull artifacts to and from your Azure Container Registry. Now ORAS can be used with ACR without additional authentication using `oras login` command.
```
Comment (Jak-MS) [inline articles/container-registry/container-registry-manage-artifact.md:60]:
```suggestion
This setup enables you to seamlessly push and pull artifacts to and from your Azure Container Registry. Now ORAS can be used with ACR without additional authentication by using `oras login` command. 
```

File: articles/container-registry/container-registry-manage-artifact.md [hunk 2]
```diff
@@ -56,36 +55,21 @@ Authenticate with the ACR, for allowing you to pull and push container images.
...
-```azurecli
-# Login to Azure
-az login
 ```
 
-### Sign in with ORAS
+This setup enables you to seamlessly push and pull artifacts to and from your Azure Container Registry. Now ORAS can be used with ACR without additional authentication using `oras login` command. 
 
-Provide the credentials to `oras login`.
+If Docker isn't available, you can utilize the AD token provided for authentication. Authenticate with your [individual Microsoft Entra identity](container-registry-authentication.md?tabs=azure-cli#individual-login-with-azure-ad) using an AD token. Always use "000..." for the `USER_NAME` as the token is parsed through the `PASSWORD` variable. Note that the token used by `az acr login` is valid for 3 hours.
```
Comment (Jak-MS) [inline articles/container-registry/container-registry-manage-artifact.md:62]:
```suggestion
If Docker isn't available, you can use the AD token provided for authentication. Authenticate with your [individual Microsoft Entra identity](container-registry-authentication.md?tabs=azure-cli#individual-login-with-azure-ad) using an AD token. Always use "000..." for the `USER_NAME` as the token is parsed through the `PASSWORD` variable. The token used by `az acr login` is valid for three hours.
```

**Pull request merged**